pkg/kapp/cmd/app/inspect.go

```diff
@@ -119,3 +122,9 @@ func (o *InspectOptions) Run() error {
 
 	return nil
 }
+
+func (o *InspectOptions) checkDeprecation(resources []resources.Resource, supportingObjs FactorySupportObjs) {
```

Comment:
Can checking for deprecation of filtered resources on a remote cluster in this manner be too much of an overhead if the number of resources increases?
It attempts to `Get()` each filtered resource.